cmd/server-main.go

```diff
@@ -233,6 +233,7 @@ func newAllSubsystems() {
 	globalConfigSys = NewConfigSys()
 
 	// Create new IAM system.
+	globalEtcdClient = nil
```

Comment:
what is this for?

Reply:
It is to reset it for tests.

Reply:
yeah hard to understand why you need it here? - you can do this in the test itself right? doing this without a comment will look slightly weird and someone might remove it.

Reply:
Fixed.